blockchain/tests/inherents.rs

```diff
@@ -47,7 +47,7 @@ fn it_can_create_batch_finalization_inherents() {
         version: 1,
         block_number: Policy::macro_block_of(2).unwrap(),
         round: 0,
-        timestamp: blockchain.state.election_head.header.timestamp + 1,
+        timestamp: blockchain.state.election_head.header.timestamp + 20000,
```

Comment:
Any reason for this 20s?

Reply:
The test has assumptions in terms of the number of reward being paid which with the new supply curve don't hold for such a short timestamp difference.